.github/workflows/continuous-integration.yml

```diff
@@ -99,6 +99,23 @@ jobs:
       - name: Test
         run: ./scripts/test
 
+  upstream:
+    name: upstream
+    runs-on: ubuntu-latest
+    # if: github.event_name != 'pull_request'
```

Comment:
Making sure this job works, then I'll uncomment the conditional so that it only runs on merge to main.

Reply:
Hmmm test didn't fail. Still futzing.

Reply:
I want this one to fail: https://github.com/stac-utils/pystac-client/actions/runs/5046890774/jobs/9053014342?pr=521